src/user/contributing.rst

```diff
@@ -48,5 +48,20 @@ We are glad to know that you would like to contribute to the ``conda-forge`` doc
    your changes.
 5. Submit a `pull request
    <https://help.github.com/articles/about-pull-requests/>`_ to the main repository proposing your changes.
+   
+Writing guidelines:
+   
+Some basic writing guidelines should be kept in mind before you start contributing:
+
+1. Identify your audience and understand their skill level.
+2. Match the technical language with the audience's skill level proficiency.
+3. Try to keep it simple. In case you have to use a lot of complicated terms, provide a glossary of key terms.
+4. Address the audience directly as the user(s).
+5. While mentioning an action, use the “command” form of the verb: “Choose an option from the menu and press Enter.”   
```

Comment:
I am not sure what our policy for stylised quotes is within the docs. IMO we should default to straight quotes (e.g. `"command"`) and let [Sphinx's `smartquotes`](https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-smartquotes) handle the rest.